src/model_repository_manager/model_lifecycle.cc

```diff
@@ -526,6 +528,7 @@ ModelLifeCycle::AsyncLoad(
     load_pool_->Enqueue([this, model_id, version, model_info, OnComplete,
                          load_tracker, is_config_provided]() {
       for (size_t retry = 0; retry <= options_.load_retry; ++retry) {
+        // TODO add here
```

Comment:
add what here?

Reply:
Initially marked as a potential place to start the counter. Forgot to clean-up the comment.
Fixed